kratos/tests/test_materials_input.py

```diff
@@ -3,6 +3,16 @@
 # Importing the Kratos Library
 import KratosMultiphysics.KratosUnittest as KratosUnittest
 import KratosMultiphysics
+try:
+    import KratosMultiphysics.FluidDynamicsApplication as KratosFluid

```

Comment:
why did you remove the StructuralMechanics here?

It is still depending on it (bcs of the const_law being used)

Reply:
Ups, true

Reply:
the question is how the test could run without it...